### DISCUSSION TITLE
Display total time in the footer under album and playlist views.

### DISCUSSION THREAD

**Discussion by IO101010** [navidrome/navidrome#3295] [Ideas]
As shown in the attached image, the total time could be displayed. 
<img width="562" alt="Screenshot 2024-09-21 at 00 04 55" src="https://github.com/user-attachments/assets/6643a4af-625c-4b87-99e4-4c66d928720c">

I find this helpful and reassuring since I often play music to accompany a task. For example, a workout or housecleaning session might take 1 hour. I'd like to know in advance that my playlist or album will last that long. I think it's unobtrusive to include it.

It could appear at the bottom of album listing as well as playlist listing. It would be nice to have on both desktop and mobile views.

**Comment by deluan** [comment]
You mean this: 🙂
![Screenshot 2024-09-20 at 7 43 23 PM](https://github.com/user-attachments/assets/2cf90278-57ab-4ad4-ae09-60bb46525740)
![Screenshot 2024-09-20 at 7 42 35 PM](https://github.com/user-attachments/assets/ec160c80-810e-41a0-96b1-d80197445cba)

**Comment by IO101010** [comment]
Yes I do 🤦🏻‍♂️Although, I’d also like it in the album view :)On 21 Sep 2024, at 00:45, Deluan Quintão ***@***.***> wrote:﻿
You mean this: 🙂
Screenshot.2024-09-20.at.7.43.23.PM.png (view on web)
Screenshot.2024-09-20.at.7.42.35.PM.png (view on web)

**Reply by deluan** [reply]
There are two screenshots, one is in a playlist ("All from Brock Berrigan") and the other is in an album ("8-bit lagerfeuer"). Or are you referring to something else?

**Reply by IO101010** [reply]
Oh, I see the other one now. Apologies, I’m on mobile and the notification email linked directly to one image. 

The only remaining case then would be portrait mode on mobile, where the time is hidden. I’m not sure what the solution would be in that case, though, except putting it at the bottom. 
![IMG_3635](https://github.com/user-attachments/assets/200eb08c-b48e-46b8-94c1-faf08b7f5d31)

**Reply by deluan** [reply]
Ah, yes. I removed it from mobile view. When working on the new UI, I'll take this into consideration. Thanks!

**Reply by IO101010** [reply]
Thank you. This is such a great application.

**Comment by IO101010** [comment]
I
